docs/distributed-plugin/README.md

```diff
@@ -6,12 +6,67 @@
 
 ## Overview
 
-This document outlines the work for how to design the plugin. 
+A plugin to ensure the consistency of multiple casbin instances in distributed situations
 
 ## Motivation
 
-Please visit https://github.com/casbin/Summer2020#3-casbin%E5%88%86%E5%B8%83%E5%BC%8F%E6%8F%92%E4%BB%B6-golang for details.
+Now the distribution of casbin is implemented based on watcher, but watcher is to let all instances reload the database after the update occurs. This is a very resource-consuming operation, so WatcherEx (https://github.com/casbin) /casbin/issues/398), you can notify other instances of the modified content, so as to achieve incremental updates and reduce the number of IOs. However, because Watcher notifies other instances after the update, it cannot be controlled for high concurrency mode, please refer to [https://github.com/casbin/casbin/issues/421](https://github.com/casbin/casbin/issues/421).
+
+
+In addition, many people hope casbin supports the master-slave mode([https://github.com/casbin/casbin/issues/402).](https://github.com/casbin/casbin/issues/402.) Their main purpose is to ensure data consistency in the case of multiple instances, so raft should be a good choice, this is a protocol that guarantees strong consistency. Through election of Leader, all other nodes are based on Leader to achieve strong consistency of the cluster. This algorithm has been adopted by many large projects such as etcd. Compared with master-slave mode, raft is a more efficient and reliable choice.
```

Comment:
`([https://github.com/casbin/casbin/issues/402).](https://github.com/casbin/casbin/issues/402.) ` is wrong.